packages/dialtone-css/lib/build/less/components/button.less

```diff
@@ -534,9 +534,115 @@
+
+        &:focus-visible {
+            z-index: var(--zi-active);
+        }
+    }
+
+    &__alpha {
+        flex-grow: 1;
+    }
+
+    &__omega {
+        // vertical divider between the two buttons
+        &::before {
+            position: absolute;
+            top: var(--button-border-radius);
+            bottom: var(--button-border-radius);
+            left: var(--dt-size-100-negative);
+            width: var(--dt-size-100);
+            background-color: var(--button-color-text); // use text color for divider line
+            opacity: var(--dt-opacity-600); // soften it
+            content: '';
+
+            // hide divider line when hovering or focusing any button within the split button
+            .d-split-btn:has(:focus-visible, :hover) & {
+                opacity: 0;
+            }
+
+            // DO NOT hide divider line for primary, outlined, or disabled buttons
+            .d-split-btn:has(:disabled) &,
+            .d-split-btn:has(.d-btn--outlined:focus-visible, .d-btn--outlined:hover) &,
+            .d-split-btn:has(.d-btn--primary:focus-visible, .d-btn--primary:hover) & {
+                opacity: var(--dt-opacity-600);
+            }
+        }
+
+        // bleed divider line to absolute top and bottom of the button
+        &.d-btn:disabled::before,
+        &.d-btn--outlined::before,
+        &.d-btn--primary::before {
+            top: var(--dt-size-100-negative);
+            bottom: var(--dt-size-100-negative);
+        }
+    }
+
+    &__alpha:disabled:not(.d-btn--outlined),
+    &__alpha:disabled:not(.d-btn--primary),
+    &__alpha.d-btn--primary,
+    &__alpha.d-btn--outlined {
+        border-right: 0;
+        border-top-right-radius: 0;
+        border-bottom-right-radius: 0;
+    }
+
+    &__omega:disabled:not(.d-btn--outlined),
+    &__omega:disabled:not(.d-btn--primary),
+    &__omega.d-btn--primary,
+    &__omega.d-btn--outlined {
+        border-left: 0;
+        border-top-left-radius: 0;
+        border-bottom-left-radius: 0;
+    }
+
+    // adjust x-padding for icon-only button at each size
+    &__omega--xs.d-btn--icon-only {
+        --button-padding-x: var(--dt-space-300);
+    }
+
+    &__omega--sm.d-btn--icon-only {
+        --button-padding-x: var(--dt-space-300);
+    }
+
+    &__omega--md.d-btn--icon-only {
+        --button-padding-x: var(--dt-space-350);
+    }
+
+    &__omega--lg.d-btn--icon-only {
+        --button-padding-x: var(--dt-space-400);
+    }
+
+    &__omega--xl.d-btn--icon-only {
+        --button-padding-x: var(--dt-space-450);
+    }
+}
+
 //  ============================================================================
 //  $   BUTTON GROUP
 //  ----------------------------------------------------------------------------
+
+.asdfasdfqwerqwer {
+    position: fixed;
+    z-index: var(--zi-notification);
+    inset: var(--dt-size-700) var(--dt-size-0) var(--dt-size-0);
+    overflow: auto;
+    background-color: var(--dt-color-surface-primary);
+}
```

Comment:
just for the temporary example I assume? haha

Reply:
Just a convenient way to show the grid over 100% of viewport.